Makefile.dep

```diff
@@ -8,6 +8,9 @@ OLD_USEPKG := $(sort $(USEPKG))
 # pull dependencies from drivers
 include $(RIOTBASE)/drivers/Makefile.dep
 
+# pull dependencies from drivers
+-include $(USEMODULE:%=$(RIOTBASE)/drivers/%/Makefile.dep)
```

Comment:
This should always process all files with a `$(sort $(wildcard))`.

There are already weird cases with `at86rf2xx` that implements multiple drivers and would break with a pattern like this.

It looks like there is not reasons to have issues with the processing order with the current definitions.